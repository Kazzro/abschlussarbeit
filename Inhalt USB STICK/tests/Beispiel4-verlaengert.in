// Auftrag IHK3 verl√§ngert
900 6
100, 100, 1, Auftrag A
210, 297, 2, Auftrag B
297, 420, 3, Auftrag C
880, 300, 4, Auftrag D
210, 297, 5, Auftrag E
100, 100, 6, Auftrag F
210, 297, 7, Auftrag G
297, 420, 8, Auftrag H
880, 300, 9, Auftrag I
210, 297, 10, Auftrag J
100, 100, 11, Auftrag K
210, 297, 12, Auftrag L
297, 420, 13, Auftrag M
880, 300, 14, Auftrag N
210, 297, 15, Auftrag O
100, 100, 16, Auftrag P
210, 297, 17, Auftrag Q
297, 420, 18, Auftrag R
880, 300, 19, Auftrag S
210, 297, 20, Auftrag T
100, 100, 21, Auftrag U
210, 297, 22, Auftrag V
297, 420, 23, Auftrag W
880, 300, 24, Auftrag X
210, 297, 25, Auftrag Y
100, 100, 26, Auftrag Z
210, 297, 27, Auftrag AA
297, 420, 28, Auftrag AB
880, 300, 29, Auftrag AC
210, 297, 30, Auftrag AD
100, 100, 31, Auftrag AE
210, 297, 32, Auftrag AF
297, 420, 33, Auftrag AG
880, 300, 34, Auftrag AH
210, 297, 35, Auftrag AI
100, 100, 36, Auftrag AJ
210, 297, 37, Auftrag AK
297, 420, 38, Auftrag AL
880, 300, 39, Auftrag AM
210, 297, 40, Auftrag AN
100, 100, 41, Auftrag AO
210, 297, 42, Auftrag AP
297, 420, 43, Auftrag AQ
880, 300, 44, Auftrag AR
210, 297, 45, Auftrag AS
100, 100, 46, Auftrag AT
210, 297, 47, Auftrag AU
297, 420, 48, Auftrag AV
880, 300, 49, Auftrag AW
210, 297, 50, Auftrag AX
100, 100, 51, Auftrag AY
210, 297, 52, Auftrag AZ
297, 420, 53, Auftrag BA
880, 300, 54, Auftrag BB
210, 297, 55, Auftrag BC
100, 100, 56, Auftrag BD
210, 297, 57, Auftrag BE
297, 420, 58, Auftrag BF
880, 300, 59, Auftrag BG
210, 297, 60, Auftrag BH