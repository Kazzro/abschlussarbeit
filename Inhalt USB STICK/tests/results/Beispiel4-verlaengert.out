// Auftrag IHK3 verlängert
Benötigte Länge: 776.8 cm
Benutzte Flaeche: 89.85 %

Positionierung der Kundenaufträge:
    0 0 100 100 - 1 - Auftrag A
    0 100 880 300 - 4 - Auftrag D
    0 400 210 297 - 2 - Auftrag B
    210 400 420 297 - 3 - Auftrag C
    630 400 210 297 - 5 - Auftrag E
    100 0 100 100 - 6 - Auftrag F
    0 697 297 210 - 7 - Auftrag G
    297 697 297 420 - 8 - Auftrag H
    594 697 210 297 - 10 - Auftrag J
    0 907 297 210 - 12 - Auftrag L
    0 1117 880 300 - 9 - Auftrag I
    594 994 100 100 - 11 - Auftrag K
    0 1417 420 297 - 13 - Auftrag M
    0 1714 880 300 - 14 - Auftrag N
    420 1417 420 297 - 18 - Auftrag R
    0 2014 297 210 - 15 - Auftrag O
    297 2014 100 100 - 16 - Auftrag P
    397 2014 297 210 - 17 - Auftrag Q
    0 2224 300 880 - 19 - Auftrag S
    300 2224 297 210 - 20 - Auftrag T
    597 2224 300 880 - 24 - Auftrag X
    300 2434 100 100 - 21 - Auftrag U
    300 2534 297 210 - 22 - Auftrag V
    300 2744 297 420 - 23 - Auftrag W
    0 3164 297 210 - 25 - Auftrag Y
    297 3164 210 297 - 27 - Auftrag AA
    507 3164 297 420 - 28 - Auftrag AB
    0 3374 297 210 - 30 - Auftrag AD
    297 3461 100 100 - 26 - Auftrag Z
    0 3584 880 300 - 29 - Auftrag AC
    0 3884 100 100 - 31 - Auftrag AE
    0 3984 880 300 - 34 - Auftrag AH
    0 4284 210 297 - 32 - Auftrag AF
    210 4284 420 297 - 33 - Auftrag AG
    630 4284 210 297 - 35 - Auftrag AI
    100 3884 100 100 - 36 - Auftrag AJ
    0 4581 297 210 - 37 - Auftrag AK
    297 4581 297 420 - 38 - Auftrag AL
    594 4581 210 297 - 40 - Auftrag AN
    0 4791 297 210 - 42 - Auftrag AP
    0 5001 880 300 - 39 - Auftrag AM
    594 4878 100 100 - 41 - Auftrag AO
    0 5301 420 297 - 43 - Auftrag AQ
    0 5598 880 300 - 44 - Auftrag AR
    420 5301 420 297 - 48 - Auftrag AV
    0 5898 297 210 - 45 - Auftrag AS
    297 5898 100 100 - 46 - Auftrag AT
    397 5898 297 210 - 47 - Auftrag AU
    0 6108 300 880 - 49 - Auftrag AW
    300 6108 297 210 - 50 - Auftrag AX
    597 6108 300 880 - 54 - Auftrag BB
    300 6318 100 100 - 51 - Auftrag AY
    300 6418 297 210 - 52 - Auftrag AZ
    300 6628 297 420 - 53 - Auftrag BA
    0 7048 297 210 - 55 - Auftrag BC
    297 7048 210 297 - 57 - Auftrag BE
    507 7048 297 420 - 58 - Auftrag BF
    0 7258 297 210 - 60 - Auftrag BH
    297 7345 100 100 - 56 - Auftrag BD
    0 7468 880 300 - 59 - Auftrag BG

Verbleibende Andockpunkte:
    880 100
    840 400
    0 697
    210 697
    630 697
    200 0
    100 100
    804 697
    297 907
    297 1117
    880 1117
    0 1417
    694 994
    594 1094
    840 1417
    420 1714
    880 1714
    297 2114
    0 2224
    694 2014
    397 2224
    897 2224
    400 2434
    597 2534
    0 3104
    597 3104
    597 2744
    300 3164
    804 3164
    297 3374
    397 3461
    297 3561
    507 3584
    880 3584
    0 3884
    880 3984
    840 4284
    0 4581
    210 4581
    630 4581
    200 3884
    100 3984
    804 4581
    297 4791
    297 5001
    880 5001
    0 5301
    694 4878
    594 4978
    840 5301
    420 5598
    880 5598
    297 5998
    0 6108
    694 5898
    397 6108
    897 6108
    400 6318
    597 6418
    0 6988
    597 6988
    597 6628
    300 7048
    804 7048
    297 7258
    397 7345
    297 7445
    507 7468
    880 7468
    0 7768
